_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation.vue

```diff
@@ -260,27 +261,22 @@
               {{ $t("cta.selectFilters") }}
             </b-button>
             <div
-              v-if="numberOfFilters && !campaignHasNoProductsFilter "
-              class="align-self-center ml-2"
+              v-if="!campaignHasNoProductsFilter"
+              class="align-self-center ml-2 font-weight-600"
             >
               {{
-                $tc(
-                  "smartShoppingCampaignCreation.filtersWithxValues",
-                  numberOfFilters,
-                  [numberOfFilters]
-                )
-              }}
-              {{ $t("campaigns.from") }}
-              {{
-                this.$store.state.smartShoppingCampaigns.dimensionChosen.name
-              }}
-              {{
-                $tc(
-                  "smartShoppingCampaignCreation.dimensionNumberOfxProducts",
-                  totalProducts,
-                  [totalProducts]
-                )
+                $t("smartShoppingCampaignCreation.xDimensionName", [
+                  $t(`smartShoppingCampaignCreation.${dimensionName}`),
+                ])
               }}
+              <span v-if="totalProducts">
+                {{ $tc(
+                  'smartShoppingCampaignCreation.filtersWithxValues',
+                  this.totalProducts,
+                  [this.totalProducts],
+                ),
+                }}
+              </span>
```

Comment:
Following my comment on _dev/src/assets/json/translations/en/ui.json, you can have one single string for the same purpose

```js
               {{ $tc(
                  'smartShoppingCampaignCreation.filtersWithxValues',
                  this.totalProducts,
                  [
                    $t(`smartShoppingCampaignCreation.${dimensionName}`),
                    this.totalProducts,
                  ],
                ),
                }}
```